src/survey/controllers/create_survey_controller.py

```diff
@@ -0,0 +1,74 @@
+from api.utils import failure_response
+from api.utils import modify_attribute
+from api.utils import success_response
+from match import match_status
+from match.models import Match
+from person.models import Person
+from rest_framework import status
+from survey.models import Survey
+
+
+# TODO(@chalo2000) Convert to celery task
+class CreateSurveyController:
+    def __init__(self, data):
+        self._data = data
+
+    def process(self):
+        # Verify that all required fields are provided
+        did_meet = self._data.get("did_meet")
+        explanation = self._data.get("explanation")
+        submitting_user_id = self._data.get("submitting_user_id")
```

Comment:
You can just get this from `request.user.id`

Reply:
(and then you can get rid of the submitting_user validation below)

Reply:
I had this in mind but obviously completely forgot lol